generators/entity-server/templates/src/main/java/package/web/rest/EntityResource.java.ejs

```diff
@@ -114,7 +114,7 @@ import static org.elasticsearch.index.query.QueryBuilders.*;<% } %>
  * REST controller for managing {@link <%= packageName %>.domain.<%= asEntity(entityClass) %>}.
  */
 @RestController
-@RequestMapping("/api")<% if (databaseType === 'sql' && !viaService) { %>
+@RequestMapping("/api")<% if (databaseType === 'sql' && !viaService && !saveUserSnapshot) { %>
```

Comment:
I'm not sure this is the right fix since saving the user snapshot should work even in NoSQL databases.